internal/sbi/consumer/ue_context_management.go

```diff
@@ -98,8 +107,9 @@ func UeCmDeregistration(ue *amf_context.AmfUe, accessType models.AccessType) (
 	switch accessType {
 	case models.AccessType__3_GPP_ACCESS:
 		modificationData := models.Amf3GppAccessRegistrationModification{
-			Guami:     &amfSelf.ServedGuamiList[0],
-			PurgeFlag: true,
+			Guami:         &amfSelf.ServedGuamiList[0],
+			PurgeFlag:     true,
+			BackupAmfInfo: ue.BackupAmfInfo,
```

Comment:
Now there is no config for backup amf, so don't fill backup amf info to UDM for now.

Reply:
Ok, I will remove it.